engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/videoSettings/VideoSettingsScreen.java

```diff
@@ -328,7 +328,7 @@ public boolean onKeyEvent(NUIKeyEvent event) {
     }
 
     public void saveSettings() {
-        logger.info("Video Settings: {}", config.renderConfigAsJson(config.getRendering()));
+        logger.atInfo().addArgument(() -> config.renderConfigAsJson(config.getRendering())).log("Video Settings: {}");
```

Comment:
Only evaluated if user closes video settings screen, so impact acceptable.
```suggestion
        logger.info("Video Settings: {}", config.renderConfigAsJson(config.getRendering())); //NOPMD
```